config/initializers/rails_admin.rb

```diff
@@ -509,6 +517,25 @@
       field :password
       field :password_confirmation
       field :role
+      group :permissions do
```

Comment:
probably we want it in three places, on the users, on the permissionables and it's own section. On the other hand, I don't think the permissions page would be extremely useful as it wouldn't have much context/order, as in a way it would just be a list of all users and all permissionable models thrown together.